screens/ReportGenerationScreen.tsx

```diff
@@ -0,0 +1,177 @@
+import { useState } from 'react';
+import { ScrollView, StyleSheet, View } from 'react-native';
+import { Button, Caption, Headline, Icon } from 'react-native-paper';
+import { DatePickerInput } from 'react-native-paper-dates';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { Appbar } from '../components/Appbar';
+import { FooterScreens, FooterView } from '../components/Footer';
+import { FullScreenActivityIndicator } from '../components/FullScreenActivityIndicator';
+import { useNucaTheme as useTheme } from '../hooks/useNucaTheme';
+import { RootStackParamList } from '../types';
+import { isSmallScreen } from '../utils/helperFunc';
+
+const getStyles = (theme: NucaCustomTheme) =>
+  StyleSheet.create({
+    saveButton: {
+      height: 64,
+      maxWidth: 340,
```

Comment:
These dimensions should be set at project level an reused in all button (or other components)
What's the approach so far? Can you extract these?
Having dimensions hardcoded per component make the app more rigid and not friendly to design changes.

Reply:
True, I think we can create a re-usable button component with this blue button (the one on the report generation screen is identical to the one on the HotspotForm).

Reply:
Perhaps calling it `NucaFormButton` makes sense, since it is only used on form-like screens (HotspotFormScreen.ts, ReportGenerationScreen.ts).

Reply:
Anyways, with the same dimensions, the re-usable button component seems to be displayed as expected on all platforms.